### PR TITLE
Done Homework1

### PR DESCRIPTION
**Pull request by konflic**
Added some prints according to task of homework 1.

### CODE REVIEW

File: index.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
+print("Start program..")
 name = input("What is your name? ")
 print(f"Hello, {name}")
+print("Done.")
```
Comment (konflic) [inline index.py:4]: You missed basic codestyle, please correct.

**Review by konflic — COMMENTED** [review]
Finished review.

**Comment by tiboAL** [comment]
i am

**Comment by Rost-777** [comment]
hi

**Comment by maridilijan** [comment]
Hi there

**Comment by dziarkachqa** [comment]
I am here?

**Comment by unitedfan1986** [comment]
hello

**Review by maridilijan — COMMENTED** [review]
Looks good!

**Review by tiboAL — COMMENTED** [review]
good program

**Pull request merged**